src/TestFramework/Coverage/JUnit/JUnitTestFileDataProvider.php

```diff
@@ -91,12 +80,26 @@ public function getTestFileInfo(string $fullyQualifiedClassName): TestFileTimeDa
             );
         }
 
+        Assert::same($nodes->length, 1);
+
         return new TestFileTimeData(
             $nodes[0]->getAttribute('file'),
             (float) $nodes[0]->getAttribute('time')
         );
     }
 
+    private static function testCaseMapGenerator(string $fullyQualifiedClassName): iterable
```

Comment:
```suggestion
    /**
     * @return iterable<string, string>
     */
    private static function testCaseMapGenerator(string $fullyQualifiedClassName): iterable
```
? (not entirely sure it's the syntax for PHPStan & Psalm)

Reply:
Yeah, I'll fix these once I get done with git bisect from #1173